GameEngine/IGameData.cs

```diff
@@ -8,8 +8,10 @@ namespace GameEngine
 {
     public interface IGameData
     {
+        string PlayerName { get; set; }
+
         List<Character> CreateAllGameCharacters();
 
-        string GetGameIntroduction();
+        string GetGameIntroduction();       
```

Comment:
Nit: we don't need to add extra spaces at the end of this method.

Reply:
I did not do that, but I will fix that.